modin/pandas/dataframe.py

```diff
@@ -901,22 +901,34 @@ def boxplot(
         )
 
     def clip(self, lower=None, upper=None, axis=None, inplace=False, *args, **kwargs):
-        raise NotImplementedError(
-            "To contribute to Pandas on Ray, please visit "
-            "github.com/modin-project/modin."
-        )
+        # validate inputs
+        if type(lower) == list or type(upper) == list:
```

Comment:
instead of `type(...) == ...` I prefer `isinstance(lower, list)`. There is also `is_list_like` that we borrow from pandas. This will be useful if you can pass a Series as `lower` or `upper`.